# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBCursor.cpp
# Commit: 9966f5d71bb2
# Full Hash: 9966f5d71bb2db2228a914743dca457dcc337d02
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Apply renamings suggested for Cursor::*Key members and related identifiers. r=ytausky,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43249
# ==============================================================================

diff -r 714d36c43204 -r 9966f5d71bb2 dom/indexedDB/IDBCursor.cpp
--- a/dom/indexedDB/IDBCursor.cpp	Wed Sep 18 06:41:23 2019 +0000
+++ b/dom/indexedDB/IDBCursor.cpp	Wed Sep 18 06:41:35 2019 +0000
@@ -391,7 +391,7 @@
 
   if (IsLocaleAware() && !key.IsUnset()) {
     Key tmp;
-    result = key.ToLocaleBasedKey(tmp, mSourceIndex->Locale(), aRv);
+    result = key.ToLocaleAwareKey(tmp, mSourceIndex->Locale(), aRv);
     if (!result.Is(Ok, aRv)) {
       if (result.Is(Invalid, aRv)) {
         aRv.Throw(NS_ERROR_DOM_INDEXEDDB_DATA_ERR);
@@ -495,7 +495,7 @@
 
   if (IsLocaleAware() && !key.IsUnset()) {
     Key tmp;
-    result = key.ToLocaleBasedKey(tmp, mSourceIndex->Locale(), aRv);
+    result = key.ToLocaleAwareKey(tmp, mSourceIndex->Locale(), aRv);
     if (!result.Is(Ok, aRv)) {
       if (result.Is(Invalid, aRv)) {
         aRv.Throw(NS_ERROR_DOM_INDEXEDDB_DATA_ERR);